octokit/emojis_test.go

```diff
@@ -19,6 +19,6 @@ func TestRootEmojisService_All(t *testing.T) {
 	assert.False(t, result.HasError())
 	var penguin = "https://github.global.ssl.fastly.net/images/icons/emoji/penguin.png?v5"
 	var metal = "https://github.global.ssl.fastly.net/images/icons/emoji/metal.png?v5"
-	assert.Equal(t, penguin, emojis["penguin"])
-	assert.Equal(t, metal, emojis["metal"])
+	assert.Equal(t, emojis["penguin"], penguin)
```

Comment:
I think these two were actually correct. The expected value is just in a variable for brevity.